resource_customizations/keda.sh/ScaledJob/health.lua

```diff
@@ -0,0 +1,24 @@
+local hs = {}
+if obj.status ~= nil then
+  if obj.status.conditions ~= nil then
+    for i, condition in ipairs(obj.status.conditions) do
+      if condition.status == "False" and condition.type == "Ready" then
+        hs.message = condition.message
+        hs.status = "Degraded"
+        return hs
+      elseif condition.status == "True" and condition.type == "Ready" then
+        hs.message = condition.message
+        hs.status = "Healthy"
+        return hs
+      elseif condition.status == "Unknown" and condition.type == "Ready" then
+        hs.message = condition.message
+        hs.status = "Degraded"
+        return hs
```

Comment:
Should we add another status check for `condition.status == "True" and condition.type == "Paused"` and set the status as `Suspended`?